README.md

```diff
@@ -28,7 +27,26 @@ Docs are available on [ReadTheDocs][rtfd]
 
 [![readthedocs](https://readthedocs.org/projects/ofcourse/badge/?version=latest)](http://ofcourse.readthedocs.org/en/latest)
 
-## Tests
+## Running An ofCourse Site
+
+### Prerequisites
+
+You'll probably want to have a python virtualenv setup for testing your
+ofCourse website. To install virtualenv simply type the following
+command into a root shell.
+
+    $ pip install virtualenv
+
+### Viewing the Site
+
+To view an ofCourse website you'll want to run the following commands.
```

Comment:
Maybe link to some explanation of venv instead of telling people to just run commands. Educational material, remember?